whiteBoarding/fizzbuzz/fizzbuzz.md

```diff
@@ -0,0 +1,5 @@
+`1st) return fizzbuzz. 2nd) return fizzbuzz but in lieu of fizz,buzz,fizzbuzz return key values from this provided object => let sounds = {
+                             cat:'meow', fizz
+                             dog:'woof', buzz
```

Comment:
this formatting needs a bit of work, build it into a code block in your markdown or at least bracket it close to the edge of the page.